### PR TITLE
Added actions for API endpoints/models

### PR DESCRIPTION
**Pull request by helomberhane**
Added API for those available at the moment (communities, passengers, medical facilities, surveillance, toll-free) models and index endpoints to actions and to keep in the store. Would update the endpoints as environment variables if we know the values, the dev endpoints are currently in there for this commit. Added index/get all routes, should make additional tickets for pagination or specific ID/get/show routes when needed.

### CODE REVIEW

**Comment by helomberhane** [comment]
Updated endpoints and updated environment variables for client id and secret.

File: store/auth.js [hunk 1]
```diff
@@ -37,5 +44,33 @@ export const actions = {
...
+  },
+
+  async getApiToken({ commit }) {
+    let clientId = ''
+    let clientSecret = ''
+    const AUTHORIZATION_KEY = Buffer.from(`${clientId}:${clientSecret}`).toString('base64')
+
+    let axiosConfig = {
+      method: 'post',
+      url: 'https://ethiopia-covid19.auth.us-east-2.amazoncognito.com/oauth2/token',
```
Comment (nabebe) [inline store/auth.js:56]: Could we have this as a parameter

Comment (helomberhane) [reply store/auth.js:56]: Yeah, I could update these to use the params/methods.

**Comment by helomberhane** [comment]
@nabebe updated axios. forgot to add/track the files for each model so added those as well

**Pull request merged**